drizzlepac/adrizzle.py

```diff
@@ -371,6 +372,7 @@ def drizFinal(imageObjectList, output_wcs, configObj,build=None,wcsmap=None,proc
         paramDict['crbit'] = configObj['crbit']
         paramDict['proc_unit'] = configObj['proc_unit']
         paramDict['wht_type'] = configObj[final_step]['final_wht_type']
+        paramDict['rules_file'] = configObj['rules_file'] if configObj['rules_file'] != "" else None
 
```

Comment:
I seem to recall the default value for a string should be an empty string and not the value None.  Same comment to line 345 above.  Making these equal to "" would also match the comment at line 171 in the astrodrizzle.help.

Reply:
You point out something which would normally not be necessary, since it is not current good practice.  However, the 'fitsblender' package was written (many years ago using pre-PEP 8 Python) with an explicit dependence on 'None' as the default value for the 'rules_file' parameter.  Therefore, in order to get the correct behavior, we need to pass in None instead of an empty string.  Could we change `fitsblender`?  Yes.  Would that open up many other new issues as that package was updated?  Almost assuredly.  Is it worth it at this time?  I would argue that at this time, it is not worth the effort.

Reply:
Agreed